.github/workflows/build.yaml

```diff
@@ -11,6 +11,11 @@ on:
         description: 'List of boards to build (comma separated identifiers)'
         required: false
         type: string
+      publish:
+        description: 'Publish build artifacts to R2 (not applicable to forks)'
+        required: true
+        type: boolean
+        default: true
```

Comment:
Could we maybe simplify this by setting this to `default: ${{ github.repository }}" == "home-assistant/operating-system" && "${{ github.event_name }}" == "release"`

Reply:
Well, it would be less cluttered then, but unfortunately not. First, this is default only for `workflow_dispatch`, i.e. on-demand build from the Actions page. Second, `default` has to be boolean, you can not use `github` context here yet.